cluster-scope/components/project-admin-rolebindings/service-catalog-k8s-plugin/rbac.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+    name: namespace-admin-service-catalog-k8s-plugin
+roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: admin
+subjects:
+    - apiGroup: rbac.authorization.k8s.io
+      kind: Group
+      name: operate-first
```

Comment:
I do not understand this cluster role, how is it different from https://github.com/operate-first/apps/blob/master/cluster-scope/components/project-admin-rolebindings/operate-first/rbac.yaml ?

You are not creating new user group here. The `service-catalog-k8s-plugin` namespace is owned by `operate-first` group, hence you should reuse the `project-admin-rolebindings/operate-first` component.